compute_endpoint/tests/unit/test_htex_regression.py

```diff
@@ -0,0 +1,95 @@
+import concurrent.futures
+import logging
+import multiprocessing
+import random
+import uuid
+
+import pytest
+from globus_compute_common import messagepack
+from globus_compute_endpoint.engines import HighThroughputEngine
+from globus_compute_sdk.serialize import ComputeSerializer
+from tests.utils import double, ez_pack_function
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def engine():
+    ep_id = uuid.uuid4()
+    engine = HighThroughputEngine(
+        label="HTEXEngine",
+        heartbeat_period=1,
+        worker_debug=True,
+    )
+    queue = multiprocessing.Queue()
+    engine.start(endpoint_id=ep_id, run_dir="/tmp", results_passthrough=queue)
+
+    yield engine
+    engine.shutdown()
+
+
+@pytest.mark.skip()
+def test_engine_submit_internal(engine):
+    "Test engine.submit with multiple engines"
+
+    param = random.randint(1, 100)
+    serializer = ComputeSerializer()
+    task_id = uuid.uuid1()
+    future = engine._submit(double, param, task_id=task_id)
+    assert isinstance(future, concurrent.futures.Future)
+
+    # The content of the future object is likely junk
+    # Do not check if the result makes sense
+    assert future.result()
```

Comment:
Ah, I see it below now.  Hmm; we definitely use the `.result()` below.  The point of this call ... is to block until result is ready?  Seems like a logical noop, since we do the same just below in `.unpack()`.  Suggest just removing this stanza.

Reply:
Done